src/osd/pg_scrubber.h

```diff
@@ -52,7 +52,10 @@ class ReplicaReservations {
 
   /// notify the scrubber that we have failed to reserve replicas' resources
   void send_reject();
-
+  
+  /// ScrubberPasskey Members
```

Comment:
The comment is irrelevant here. We do not care about the mechanism at this specific point - just what this member does.